cou/apps/auxiliary.py

```diff
@@ -183,6 +183,14 @@ class RabbitMQServer(AuxiliaryApplication):
     # COU changes to 3.9 if the channel is set to 3.8
     multiple_channels = True
 
+    def _check_auto_restarts(self) -> None:
+        """No-op, skip check auto restarts option.
+
+        This method override the parent class's `_check_auto_restarts()` method
+        because the parent class's will raise an `ApplicationError` if
+        `enable-auto-restarts` is `True`.
+        """
```

Comment:
The pattern of inheritance we have in cou is not nice - we shouldn't have to add no-op override methods. :confused: 

Either way, could you move this back to its original location to avoid the diff?

Reply:
Resolving: addressed in the previous PR